ap_src/ap_app/utils/switch_permissions.py

```diff
@@ -4,6 +4,8 @@
 from .teams_utils import get_users_sharing_teams
 from .user_profile import get_user_id_from_username, get_userprofile_id_from_user_id
 
+NO_ERRORS = True
```

Comment:
This is not really necessary and can just be removed. If you wanted this variable just to improve readability, it would be better to stick it inside each function.

Reply:
It was to improve readability and it has been placed inside each function (local scope) with commit dae1d0c